java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/SessionBytesImpl.java

```diff
@@ -10,6 +10,13 @@
 
   private final EnvelopeEncryption<D> envelopeEncryption;
 
+  /**
+   * Creates a new {@code SessionBytesImpl} instance using the provided {@link EnvelopeEncryption} object. An
+   * implementation of {@link Session} that encrypts a payload of type byte[].
+   *
+   * @param envelopeEncryption An implementation of
+   *                           {@link com.godaddy.asherah.appencryption.envelope.EnvelopeEncryptionBytesImpl}.
```

Comment:
this could be any implementation. Not necessary EnvelopeEncryptionBytesImpl. Need to reword it a little